linode/apl-core
# feat: argo app removal

This PR ensures that when an app is disabled in the values repo by setting the value of `apps.<app-name>.enabled` to `false`, its corresponding argo application and associated resources are removed from the cluster.

How to test:
1. enable loki via apl console
2. deploy changes
3. wait until the application is deployed
4. go to values repo and set `apps.loki.enabled: false`
5. Observe that corresponding ArgoCd application is removed
6. Observe that corresponding k8s resources are removed
